main.go

```diff
@@ -8,23 +8,17 @@ import (
 	"os"
 )
 
-// Generate docs for website
-//go:generate go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
-
-var ProviderHost = "https://api.instaclustr.com"
-
 const ProviderName = "instaclustr"
 
 func main() {
 
-	swaggerHostOverride := os.Getenv("IC_TF_PROVIDER_SWAGGER_HOST_OVERRIDE") //needed to target locally run API to generate docs for testing/preview
+	providerHost := os.Getenv("IC_API_URL") //needed to target locally run API to generate docs for testing/preview
```

Comment:
won't this always overwrite what was passed in from the makefile?

Reply:
No it won't, the URL isn't baked in on the built binary. The built binary will only override when it detects the `IC_API_URL` environment variable.